catkit/hardware/iris_ao/segmented_dm_command.py

```diff
@@ -157,7 +163,7 @@ def get_segment_list(self):
         This list is passed to Poppy to help create the aperture.
 
         :param num_rings: int, The number of rings in your pupil
-        :return: list, the list of segments
+        :return: list, the list of segments # TODO: the list of segment *names* (as e.g. in the GUI)?
```

Comment:
This is not segment names. This is the total number of segments in a pupil given the number of rings. Remove #TODO. Clarify parameters description as you see necessary